project/LibDependencies.scala

```diff
@@ -29,15 +28,15 @@ object LibDependencies {
   ) ++ shared
 
   val play29 = Seq(
-    "com.typesafe.play"      %% "play"               % play29Version,
-    "com.typesafe.play"      %% "filters-helpers"    % s"$play29Version-M6",
-    "com.typesafe.play"      %% "play-test"          % play29Version % Test,
-    "org.scalactic"          %% "scalactic"          % "3.2.3"       % Test,
-    "org.scalatest"          %% "scalatest"          % "3.2.3"       % Test,
-    "org.scalacheck"         %% "scalacheck"         % "1.14.0"      % Test,
-    "org.scalamock"          %% "scalamock"          % "4.3.0"       % Test,
-    "com.vladsch.flexmark"    % "flexmark-all"       % "0.64.8"      % Test,
-    "org.scalatestplus.play" %% "scalatestplus-play" % "6.0.0"       % Test
+    "com.typesafe.play"      %% "play"                 % play29Version,
+    "com.typesafe.play"      %% "play-filters-helpers" % play29Version,
```

Comment:
made this match what we have in play-frontend-hmrc - not sure what the M6 represented here, but assuming it was a early version with compatibility before the new package was created for 2.9 onwards as play-filters-helpers (with 3.0 also moving it to the new org too)

Reply:
removed support for play28 and play29 now because no-one using them anymore, everyone on play30 so this is no longer relevant